app/scripts/controllers/mmi-controller.test.ts

```diff
@@ -98,13 +98,6 @@ describe('MMIController', function () {
           'NetworkController:infuraIsUnblocked',
         ],
       }),
-      state: {
-        providerConfig: {
-          type: NETWORK_TYPES.SEPOLIA,
-          chainId: CHAIN_IDS.SEPOLIA,
-          ticker: TEST_NETWORK_TICKER_MAP[NETWORK_TYPES.SEPOLIA],
-        },
-      },
```

Comment:
why this hasn't been replaced with the new state and has instead simply been deleted?

Reply:
I think just because it passed without it.  But I put back an explicit sepolia mock in https://github.com/MetaMask/metamask-extension/pull/26150/commits/ef2eff3cc14deef6b28a9e92bab089abde0a163e